goals.md

```diff
@@ -1,19 +1,23 @@
 # Goals for the EUREC4A metadata repository
 
-In order to find a good design for the metadata structure, a set of goals which should be accomplished with the metadata must be established.
-Based on these goals and available information about platforms, instruments and the like, a prototype for the metadata structure will be established.
-This document is explicitly meant to be updated, comments are always welcome.
+To design the metadata structure, a set of goals that this metadata should serve is to be established. Based on these goals and available information about platforms, instruments and the like, a prototype for the metadata structure will be established. This document is explicitly meant to be updated, comments are always welcome.
 
 ## direct goals
 The achievement of these goals should be easily decidable by a yes / no answer.
 
-* get a complete list of platform <-> PI associations
-* list of measured quantities
-* contact information for data usage
-* provide unique names / ids to platforms and instruments to facilitate data exchange and communication
-* source for campaign definitions (e.g. flight circles)
-* produce an [intake](https://intake.readthedocs.io/en/latest/index.html) catalog for EUREC4A
-* produce [GCMD Keywords](https://earthdata.nasa.gov/earth-observation-data/find-data/gcmd/gcmd-keywords). See also [here](https://gcmd.nasa.gov/search/Keywords.do#keywords) and [here](https://earthdata.nasa.gov/earth-observation-data/find-data/gcmd/gcmd-keywords)
+* establish ownership and responsibility for measurements, e.g.,
+  - association with a PI
+  - institutional ownership, if applicable
+  - contact for usage questions
+  - provide means of acknowledging use of data.
+* ability to localize measurement (time, place, instrument, quantity), e.g.,
+  - through unique names / ids to platforms and instruments
+  - list of measured quantities
+  - platform that hosted measurement
+* ability to trace version history
```

Comment:
Is `ability to trace version history` related to version history of the metadata or should the metadata aid in tracing version history of the measurement data?